install/mysql/glpi-empty.sql

```diff
@@ -2431,7 +2435,6 @@ CREATE TABLE `glpi_domains` (
   `users_id_tech` int(11) NOT NULL DEFAULT '0',
   `groups_id_tech` int(11) NOT NULL DEFAULT '0',
   `others` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-  `is_helpdesk_visible` tinyint(1) NOT NULL DEFAULT '1',
```

Comment:
Should be dropped from update if present

Reply:
Also, maybe add a note in the PR (and to future release notes :D); to tell users a forced upgrade will be required for this patch to be applied.